corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -57,26 +57,72 @@ hqDefine('analytix/js/hubspot', [
             formId: "38980202-f1bd-412e-b490-f390f40e9ee1",
             target: "#get-demo-cta-form-content",
             css: "",
+            onFormReady: function () {
+                var kissmetrics = hqImport('analytix/js/kissmetrix'),
```

Comment:
This is a requirejs module, so add this to the dependencies at the start of the file instead.